src/equations/compressible_euler_3d.jl

```diff
@@ -742,6 +742,36 @@ end
 end
 
 
+# Rotate normal vector to x-axis; normal, tangent1 and tangent2 need to be orthonormal
+@inline function rotate_to_x(u, normal, tangent1, tangent2, equations::CompressibleEulerEquations3D)
```

Comment:
IIRC, elsewhere you call these `normal_vector`, `tangent_vector1` etc. I'd prefer a consistent naming. Although I am a big fan of being descriptive, maybe in this case it is even ok to drop the suffix `vector`? I don't know :thinking: